cmake/RunSimulation.cmake

```diff
@@ -118,11 +118,12 @@ if(BUILD_FORTRAN)
     if("${BASEMENT_RESULT}" GREATER -1)
       # Copy files needed for Basement
       file(COPY "${SOURCE_DIR}/idd/BasementGHT.idd" DESTINATION "${OUTPUT_DIR_PATH}")
+      file(COPY "${OUTPUT_DIR_PATH}/in.idf" DESTINATION "${OUTPUT_DIR_PATH}/BasementGHTIn.idf")
```

Comment:
Attempt to get the basement program working (it still isn't)

Reply:
I'm getting errors such as 
```
IP: IDF line~5384 Did not find “Output:Variable” in list of Objects;
 Output:PreprocessorMessage,GroundTempCalc - Basement,Severe,
IP: The IDF file has no records.;
 Output:PreprocessorMessage,GroundTempCalc - Basement,Severe,
IP: Other miscellaneous errors found in input;
   **   ~~~   ** Possible Invalid Numerics or other problems
 Output:PreprocessorMessage,GroundTempCalc - Basement,Fatal,```

Reply:
Oh ok. https://github.com/NREL/EnergyPlus/blob/a119feb883d2f69e4eb462811d0628352c2de544/testfiles/SingleFamilyHouse_HP_Slab.idf#L2623-L2626